spec/rails_helper.rb

```diff
@@ -2,9 +2,9 @@
 
 ENV['RAILS_ENV'] = 'test'
 
-require "active_admin/dependency"
+require_relative "../tasks/test_application"
 
-require_relative "../tmp/rails/rails-#{ActiveAdmin::Dependency.rails_version}/config/environment"
+require "#{ActiveAdmin::TestApplication.new.full_app_dir}/config/environment.rb"
```

Comment:
I had a concern here possibly with the defaults but both would be ones we'd want anyway since this is always running in the test environment (along with the `features/support/env.rb` file) so all good.